src/components/sprite-selector-item/sprite-selector-item.css

```diff
@@ -20,7 +20,6 @@
     cursor: pointer;
 
     user-select: none;
-    touch-action: none;
```

Comment:
Removing this line because it was added to prevent scrolling by touching the tiles, which is the opposite of what we want now. Now the scroll prevention is handled dynamically by selectively preventing default on certain touch moves.